src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature

```diff
@@ -0,0 +1,44 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Validate Creation for Absence Event Category Descriptors
```

Comment:
"Absence Event Category" is the specific example, but the feature is more general than this. We are validating the behavior when a client application tries to issue a POST request.